azure-pipelines.yml

```diff
@@ -498,7 +502,34 @@ stages:
       displayName: Call repox
       steps:
       - checkout: none
-
+      - powershell: |
+          $isPr=$env:BUILD_REASON -eq 'PullRequest'
+          $repoPrefix = 'sonarsource-public'
+          $it = Get-Date -Format s
+          if ($isPr)
+          {
+            $targetRepo = $repoPrefix + '-dev'
+            $repoxStatus = 'it-passed-pr'
+          } 
+          else
+          {
+            $targetRepo = $repoPrefix + '-builds'
+            $repoxStatus = 'it-passed'
+          }
+          Write-Host "##vso[task.setvariable variable=targetRepo;]$targetRepo"
+          Write-Host "##vso[task.setvariable variable=repoxStatus;]$repoxStatus"
+          Write-Host "##vso[task.setvariable variable=it;]$it"
+        displayName: Calculate promotion values
+      - task: JFrogBuildPromotion@1
```

Comment:
The task does not support setting the properties. Therefore we miss setting "IT":
https://github.com/SonarSource/sonar-scanner-msbuild/pull/2173/files#diff-7915b9b726a397ae7ba6af7b9703633d21c031ebf21682f3ee7e6a4ec52837a5R574
I do not know if this is a problem and if so we may use the [Setting / Deleting properties on files in Artifactory](https://github.com/jfrog/jfrog-azure-devops-extension?tab=readme-ov-file#setting--deleting-properties-on-files-in-artifactory) task for this.

Reply:
Hello @martin-strecker-sonarsource .
From my side the new task is LGTM.
Let's ask the REs, though, if they have any idea why this is the case.

Reply:
Julien replied:

> This property used to be set when the QA was running on Jenkins: https://github.com/SonarSource/jenkinsfile-library/blob/8d0eb9020713c2bd9e9886fc09776e12ecf90575/vars/repoxPromoteBuild.groovy#L36
It has not been set on many projects since we migrated to Cirrus, so I think you can safely drop it.
[repoxPromoteBuild.groovy](https://github.com/SonarSource/jenkinsfile-library/blob/8d0eb9020713c2bd9e9886fc09776e12ecf90575/vars/repoxPromoteBuild.groovy)
        "it" : [ "${formatTimestamp(System.currentTimeMillis())}" ]
        
   I guess we can move on.